data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/TopicConfig.java

```diff
@@ -24,12 +25,12 @@ public class TopicConfig {
     static final String DEFAULT_AUTO_OFFSET_RESET = "earliest";
     static final Duration DEFAULT_THREAD_WAITING_TIME = Duration.ofSeconds(5);
     static final Duration DEFAULT_MAX_RECORD_FETCH_TIME = Duration.ofSeconds(4);
-    static final Integer DEFAULT_FETCH_MAX_BYTES = 52428800;
+    static final String DEFAULT_FETCH_MAX_BYTES = "50mb";
     static final Integer DEFAULT_FETCH_MAX_WAIT = 500;
-    static final Integer DEFAULT_FETCH_MIN_BYTES = 1;
+    static final String DEFAULT_FETCH_MIN_BYTES = "1b";
     static final Duration DEFAULT_RETRY_BACKOFF = Duration.ofSeconds(10);
     static final Duration DEFAULT_RECONNECT_BACKOFF = Duration.ofSeconds(10);
-    static final Integer DEFAULT_MAX_PARTITION_FETCH_BYTES = 1048576;
+    static final String DEFAULT_MAX_PARTITION_FETCH_BYTES = "1mb";
```

Comment:
Should we make it configurable at byte level instead>? to be at par with kafka config.